backend/src/API/devPortfolioAPI.ts

```diff
@@ -1,8 +1,13 @@
+import { DateTime } from 'luxon';
 import DevPortfolioDao from '../dao/DevPortfolioDao';
 import PermissionsManager from '../utils/permissionsManager';
 import { PermissionError, BadRequestError } from '../utils/errors';
 import { validateSubmission, isWithinDates } from '../utils/githubUtil';
 
+function zonedTime(timestamp: number, ianatz = 'America/New_York') {
```

Comment:
Can we keep the functions consistent here? The other functions don't use the `function` keyword so maybe rewrite this one also as an anonymous function too.

Reply:
makes sense :)